flink-libraries/flink-gelly-examples/src/main/java/org/apache/flink/graph/Runner.java

```diff
@@ -103,6 +113,18 @@
 		.addClass(PageRank.class)
 		.addClass(TriangleListing.class);
 
+	private ParameterTool parameters;
+
+	private BooleanParameter disableObjectReuse = new BooleanParameter(this, "__disable_object_reuse");
+
+	private StringParameter jobDetails = new StringParameter(this, "__write_job_details")
```

Comment:
rename parameter to `__job_details_path` ?  `__write_job_details_to_path`?

and rename field to `jobDetailsPath`.

Now it is unclear what this variable represents without looking at the usages.